### DISCUSSION TITLE
Safari Window is remotely controlled by an automated test.

### DISCUSSION THREAD

**Discussion by Uno-count** [seleniumbase/SeleniumBase#3150] [Q&A] [answered]
I was using a dialog box in my test automation, but whenever I interact with it, a prompt unexpectedly appears from the Safari driver. Do you have a solution for this?

![image](https://github.com/user-attachments/assets/33806ea0-0a6c-4812-8e6b-6560dc04a3c1)

**Comment by mdmintz** [comment, marked as answer]
Safari doesn't let you continue automating if you perform manual actions during your session.
You can either do all automation, or switch fully to manual (without being able to switch back).
https://discussions.apple.com/thread/254170864?sortBy=rank
